app.js

```diff
@@ -1,45 +1,164 @@
+//autmoatically evaluatec to being 'true'
 
-// 10 - 
+//prediction: false, the datatypes do not match
+// console.log('true' === true)
 
+//prediction: true
+// console.log(10 % 2 === 0);
 
-//// Task 3 - Conditionals Practice 
+//prediction: false // the remainder of 21 / 2 = 1
+// console.log(21 % 2 === 0);
+
+//prediction: true
+// console.log(21 % 2 !== 0);
+
+//prediction: true
+// console.log(21 % 2 === 1);
+
+//prediction: true 
+// console.log(12 % 3 === 0);
+
+//prediction:true 
+// console.log(9 % 3 === 0)
+
+//prediction:false
+// console.log(14 % 3 === 0);
+
+//prediction: 'n'
+// console.log('new york'[0]);
+
+//prediction: 'e'
+// console.log('new york'[1]);
+
+//prediction: NaN
+// console.log('san francisco'[2 * 3]); //'a' is returned 
+//this is because JS operates on the indexes first before looking at the string to evaluate
+//additionally JS considers spaces made in a string to be a character istself, 2*3 =6 
+//the 6th character in 'san francisco' including the space is 'a' in francisco 
+
+//prediction: 'engIneering' 
+// console.log('engineering'[3].toUpperCase()); 
+//JS only outputs the single letter at the index number
+// I is returned 
+
+//prediction: since "G" is not technically in the string it will return undefined
+//console.log('engineering'.indexOf('G')); 
+//JS returns -1 as the starting index for characters not found in the string, and adds 
+//additional characters not found in string as negative numbers.
+
+//prediction: I'm not sure, maybe 4, 5, 6 ,7
+//console.log('engineering'.indexOf('neer'));
+//JS will interpret the index value of a part of a string at the starting index of the entire string
+//in this case the starting index of 'neer' is 4, so JS evaluates the whole string as having an index of 4
+
+//prediction: true //index of r is 7 and 7 > -1
+//console.log('engineering'.indexOf('r') > -1);
+
+//prediction:false // 'x' is not the same datatype as -1
+//console.log('science'.indexOf('x') === -1);
+//JS evaluates this as true regardless of data type
+
+// List 3 expressions that surprised you or gave unexpected results, why?
+
+//"san francisco'[2 * 3]" Since indexes represent parts of a string I didnt know what JS would do. 
+//But I realised that it would evaluate the numbers first 2*3=6 and then use the result as the index number
+
+//'engineering'.indexOf('neer') --> I was not sure how JS would evaluate a partial string, but it seems like it evaluates it using the
+//index number the partial string starts on
+
+//'science'.indexOf('x') === -1  --> this surprised me because I was expecting JS to not compare a string with a num datatype, I assumed the answer would be false
+
+// List 2 exprsssions you felt comfortable interpreting at a glance.
+//true && !(false || false)
+//'cat' + 'dog'
+
+// List 1 expression you are still 'shaky' on, why?
+//'science'.indexOf('x') === -1  I'm still not really sure how this works
```

Comment:
`-1` means it does not exist. So in this example, there is n `x` in `science` so the indexOf method returns `-1`